shopBuddy/shopBuddy/ProfileViewController.m

```diff
@@ -22,8 +22,7 @@ - (void)viewDidLoad {
     // Do any additional setup after loading the view.
 }
 - (IBAction)didTapLogout:(id)sender {
-    [PFUser logOutInBackgroundWithBlock:^(NSError * _Nullable error) {
-    }];
+    [PFUser logOutInBackgroundWithBlock:^(NSError * _Nullable error) {nil;}];
```

Comment:
I was suggesting passing `nil` for the block param.

`[PFUser logOutInBackgroundWithBlock:nil];`